packages/account/src/BiconomySmartAccount.ts

```diff
@@ -23,13 +25,15 @@ import {
   SmartAccountsResponse,
   SCWTransactionResponse
 } from '@biconomy/node-client'
-import { ENTRYPOINT_ADDRESSES, BICONOMY_FACTORY_ADDRESSES } from './utils/Constants'
+import { ENTRYPOINT_ADDRESSES, BICONOMY_FACTORY_ADDRESSES, BICONOMY_IMPLEMENTATION_ADDRESSES } from './utils/Constants'
 
 export class BiconomySmartAccount extends SmartAccount implements IBiconomySmartAccount {
-  private factory: SmartAccountFactory_v100
+  private factory!: SmartAccountFactory_v100
   private nodeClient: INodeClient
-  private accountIndex!: Number
+  private accountIndex!: number
   private address!: string
+  private smartAccountInfo!: ISmartAccount
+  private isInited!: boolean
```

Comment:
Rename `isInited` to `isInitialized`

Reply:
can just keep boolean called initialized